src/jsony.nim

```diff
@@ -116,8 +116,8 @@ proc parseHook*(s: string, i: var int, v: var SomeSignedInt) =
       parseHook(s, i, v2)
       try:
         v = type(v)(v2)
-      except:
-        error("Number type to small to contain the number.", i)
+      except CatchableError:
```

Comment:
Is this error actually CatchableError? I have not tested. 

Could you make a test case for this and see what the behavior is?